content/03.use-cases-a.md

```diff
@@ -47,3 +47,8 @@ There are (as always) no "points" awarded for using ALL GitHub materials ALL the
 
 Matthew D. Beckman, Mine Çetinkaya-Rundel, Nicholas J. Horton, Colin W. Rundel, Adam J. Sullivan & Maria Tackett (2021) Implementing Version Control With Git and GitHub as a Learning Objective in Statistics and Data Science Courses, Journal of Statistics and Data Science Education, 29:sup1, S132-S144, DOI: 10.1080/10691898.2020.1848485
 
+
+Github is a valuable tool/platform to support both learning and teaching as a submission platform (to host their work and submit their course assignments and
```

Comment:
Think it could be worth it to quickly list the ways this is helpful (i.e. "...a way to host course content, through basic GitHub, the organizations feature, and GitHub classroom."

Reply:
It also seems like this should be integrated with the paragraph above.  I think it's a good revision or addition to lines 31–32

Reply:
> It also seems like this should be integrated with the paragraph above. I think it's a good revision or addition to lines 31–32

@Aariq  Agreed.

Reply:
> Think it could be worth it to quickly list the ways this is helpful (i.e. "...a way to host course content, through basic GitHub, the organizations feature, and GitHub classroom."

@colebrookson thank you so much for your valuable comment. I think it is edited by now and done! Could you please double-check?